docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md

```diff
@@ -173,7 +174,7 @@ Gallery](https://greatexpectations.io/expectations/expect_table_column_count_to_
 
 ## Examples
 
-**GX Cloud** provides a visual interface to create and run schema validation workflows. The GX Cloud workflow for validating data schema is to create a Data Asset, define Expectations, run a Validation, and review Validation Results.
+**GX Cloud** provides a visual interface to create and run schema validation workflows. The GX Cloud workflow for validating data schema is to create Data Assets with automatically generated Expectations, manually define additional Expectations, run Validations, and review Validation Results.
```

Comment:
Should `validating data schema` be `validating a data schema`?

Reply:
good question! I'm going to change this to "validating data schemas" (plural) so I can still say "automatically generated Expectations" (plural) to future proof this a bit so it works for the current state of there only being one Expectation that can be autogenerated and will continue to work after we add more.

Reply:
```suggestion
**GX Cloud** provides a visual interface to create and run schema validation workflows. The GX Cloud workflow for validating data schemas is to create Data Assets with automatically generated Expectations, manually define additional Expectations, run Validations, and review Validation Results.
```